app/src/main/AndroidManifest.xml

```diff
@@ -32,6 +32,18 @@
         <activity
             android:name=".view.EventInfoActivity" />
 
+        <activity android:name=".view.LoginActivity" >
```

Comment:
Please lock the orientation of this activity to portrait mode.

Reply:
Should be added when he merges with development